app/build.gradle

```diff
@@ -37,6 +37,20 @@ def initApiKeys(defaultConfig) {
     }
 }
 
+String getVersionBuild() {
+    def buildNumberFromCi = System.getenv("CIRCLE_BUILD_NUM")
+
+    if (buildNumberFromCi != null) {
+        return VERSION_BUILD as Integer + buildNumberFromCi.toInteger()
+    } else {
+        return VERSION_BUILD
+    }
+}
+
+String getBuildAppFileName() {
+    """p2p_wallet_${getVersionBuild()}_${VERSION_NAME}_${taskNumber()}"""
```

Comment:
I don't think we need tasknumber here. 
Let's make the following format: 

"""P2P_Wallet_${VERSION_NAME} (${getVersionBuild()})"""

And can we add build type as well?

Reply:
well, that would be very painful to do because we need to create a separate GRADLE task and get build type there. 

also, about taskNumber, how should the testers understand which branch was built?

Reply:
in release notes